package.json

```diff
@@ -78,12 +78,12 @@
     "karma-jasmine": "4.0.1",
     "karma-jasmine-html-reporter": "1.7.0",
     "lint-staged": "^10.5.3",
-    "ng-packagr": "12.1.1",
+    "ng-packagr": "13.0.6",
     "prettier": "2.3.2",
     "rxjs": "6.6.3",
```

Comment:
Do we want to bump this to v7 or would sticking at v6 be better to catch errors that might occur from those still on v6?
As far as I know Angular supports both v6 and v7.

Reply:
We can stick with v6 for now.